admin_tests/metrics/test_utils.py

```diff
@@ -181,3 +183,34 @@ def test_get_all_user_count(self):
     def test_get_unregistered_users(self):
         count = get_unregistered_users()
         nt.assert_equal(count, 1)
+
+
+class TestRenderToCSVResponse(AdminTestCase):
+
+    def setUp(self):
+        super(TestRenderToCSVResponse, self).setUp()
+        self.app = TestApp(test_app)
+        Node.remove()
+        time_now = get_previous_midnight()
+        NodeFactory(category='project', date_created=time_now)
+        NodeFactory(category='project',
+                    date_created=time_now - timedelta(days=1))
+        last_time = time_now - timedelta(days=2)
+        NodeFactory(category='project', date_created=last_time)
+        NodeFactory(category='project', date_created=last_time)
+        get_days_statistics(last_time + timedelta(seconds=1))
+        self.time = time_now + timedelta(seconds=1)
+
+        self.FULL_PERSON_CSV_NO_VERBOSE = ['id,users,delta_users,unregistered_users,projects,delta_projects,'
+                                           'public_projects,delta_public_projects,registered_projects,'
+                                           'delta_registered_projects,date\r',
+                                           '1,0,0,0,0,0,0,0,0,0,2016-09-25 00:00:01.000001\r', '']
```

Comment:
I'm not entirely sure what's going on in this test. I would expect that, since you're generating data with dynamic dates and such, that at least some of this data would need to be pulled from the models that you built in the factories.

Also, if you were to add creators to the nodes, would you get some different values for projects, public_projects, etc?